app/models/manageiq/providers/ansible_tower/inventory/parser/automation_manager.rb

```diff
@@ -34,15 +33,13 @@ def configuration_scripts
       inventory_object.variables = job_template.extra_vars_hash
       inventory_object.inventory_root_group = persister.inventory_root_groups.lazy_find(job_template.inventory_id.to_s)
 
-      authentications = []
+      inventory_object.authentications = []
```

Comment:
yeah, so now the scanning is as a separate step and you can do   inventory_object.authentications = [] and inventory_object.authentications << xy

before, the dependency scan was done on assignment, so inventory_object.authentications = [] would do the scan, but inventory_object.authentications << xy would not, since you access the [] directly